docs/data-apps/attribution-modeling/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Attribution Modeling"
+sidebar_position: 3
+sidebar_label: "Attribution Modeling"
+---
+
+:::caution
+
+This data app is currently in Private Preview and features may change without notice. 
+
+:::
+
+In today's increasingly complex digital world, users often take multi-channel journeys before converting. Assigning credit across multiple touchpoints is vital to getting an accurate picture of the efficacy of your marketing channels, yet requires merging disparate datasets and running complex calculations.
+
+Our **Attribution modeling** app lowers the barrier to entry for your marketing team through the following features:
+
+- Incremental SQL model in your warehouse for cost-effective computation
+- Choice of first-touch, last-touch, linear and positional methods, with additional filters and transforms available
+- Reports for conversions, revenue, spend and ROAS per channel and campaign
+- Specify your own advertising spend table to generate a Return on Ad Spend analysis
+- Intermediate tables that you can build your own attribution models on top of
```

Comment:
Is this true about the app? Or just the models?

Reply:
This was taken from the marketing approved metadata, so I thought the package is part of the app "experience" due to the ootb nature? But I agree it is confusing in this context.

Reply:
I have added a reference to the package in brackets to make more sense and avoid having to delete things